client/lib/perfmon/index.js

```diff
@@ -152,7 +152,7 @@ function recordPlaceholders( mutation ) {
 		return;
 	}
 
-	each( nodes, function ( node ) {
+	nodes.forEach( function ( node ) {
```

Comment:
Should this also use `Array.from`? `nodes` is a `NodeList` coming from `mutation.addedNodes`, from the Mutation Observer DOM API.

Reply:
Good point, I don't think it would hurt to have it just in case 👍

Reply:
Done in 7a8a4372157e28cf292a0334dbe0837c5da51025